lib/webauth.js

```diff
@@ -207,6 +210,36 @@ module.exports = function(authManager, cookiePort, isSecurePort) {
     return;
   });
 
+  const _passwordResetInner = Promise.coroutine(function*(req, res, type) {
+    const handlers = getRelevantHandlers(authManager, req.body);
```

Comment:
More correct to just use authManager.getAllHandlers() here since it is deliberate rather than accidental, since the behavior you observe is I think accidental currently.